pkg/cli/parse.go

```diff
@@ -26,12 +26,23 @@ func locationName(l string) string {
 }
 
 func (e *Parse) Run(_ *cobra.Command, args []string) error {
-	input, err := input.FromFile(args[0])
+	var (
+		content string
+		err     error
+	)
+
+	// Attempt to read the file first, if that fails, try to load the URL. Finally,
+	// return an error if both fail.
+	content, err = input.FromFile(args[0])
 	if err != nil {
-		return err
+		log.Debugf("failed to read file %s (due to %v) attempting to load the URL...", args[0], err)
+		content, err = input.FromURL(args[0])
```

Comment:
It's possible to get `"", nil` here. This indicates that `args[0]` isn't a URL or GitHub repo.

I think it makes sense to return the original error in this case. What do you think?

Reply:
I agree and will add that.